src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -232,17 +240,17 @@ private void setParameterMap(
 
     this.checkAndUpdate(
         MAX_CLIENT_LAG,
-        isIcebergMode ? MAX_CLIENT_LAG_ICEBERG_MODE_DEFAULT : MAX_CLIENT_LAG_DEFAULT,
+        isIcebergMode() ? MAX_CLIENT_LAG_ICEBERG_MODE_DEFAULT : MAX_CLIENT_LAG_DEFAULT,
```

Comment:
do we have a test that makes sure:
1. iceberg mode defaults are what we expect (max chunks = 1 / max client lag = 30 / compression = zstd / etc.)
2. validates that max chunks cannot be set to anything but 1 by the customer

Reply:
We have 1 in `ParameterProviderTest.withDefaultValues`, added test for 2.